packages/liquidator/test/Liquidator.js

```diff
@@ -137,9 +155,11 @@ contract("Liquidator.js", function(accounts) {
 
     liquidator = new Liquidator(
       spyLogger,
+      oneInch,
```

Comment:
Can you make `oneInch` undefined as the base case for all the tests, and then just pass it in for the one test you added?